smf/sled-agent/gimlet-standalone/config-rss.toml

```diff
@@ -22,25 +23,82 @@ ntp_servers = [ "ntp.eng.oxide.computer" ]
 dns_servers = [ "1.1.1.1", "9.9.9.9" ]
 
 # Delegated external DNS zone name
+#
+# The rack provides separate external API and console endpoints for each Silo.
+# These are named `$silo_name.sys.$external_dns_zone_name`.  For a Silo called
+# "eng" with delegated domain "oxide.example", the API would be accessible at
+# "eng.sys.oxide.example".  The rack runs external DNS servers that serve A/AAAA
+# records for these DNS names.
+#
+# In a real deployment, the operator would own this DNS domain and delegate it
+# to the rack.  That means adding glue records in their DNS servers for the
+# parent domain that point to the rack's external DNS servers.
+#
+# In development, we usually use a fake domain here and configure clients to use
+# the rack's external DNS server as their normal DNS server.  Programmatically,
+# we do this with a custom reqwest Resolver.  Using curl or the Oxide CLI, you
+# can do this with the `--resolve` option.  In practice, people often skip DNS
+# altogether (which also means skipping TLS).
 external_dns_zone_name = "oxide.test"
 
 # Initial TLS certificates for the external API
 #
+# Each entry here is really a TLS certificate chain.  You typically only need
+# one entry here.  Multiple are supported primarily for rotating them.
+#
+# The certificates provided here will be used for the "recovery" Silo that's
+# created during initial setup with Wicket.  Immediately after that setup, users
+# will go to https://recovery.sys.$external_dns_zone_name to reach the console
+# to finish initial setup.  The rack will serve one of these TLS certificates at
+# that time.  Once in the console, the user would generally create another Silo
+# with its own certificate(s).
+#
 # For the structure of these certificates, see `Certificate` in the Nexus
-# internal API.  In practice, it can be unwieldy to put them here.  You can also
-# specify a certificate by including the certificate chain and private key in
-# PEM-format files called "initial-tls-cert.pem" and "initial-tls-key.pem",
-# respectively, in the same place as this configuration file.
+# internal API.  In practice, it's unwieldy to put them in this file.  Instead,
+# you can specify a certificate by including the certificate chain and private
+# key in PEM-format files called "initial-tls-cert.pem" and
+# "initial-tls-key.pem", respectively, in the same place as this configuration
+# file.
```

Comment:
worth adding an example in the comment here, of what it looks like when this field is populated?

Reply:
I don't think so because I cannot imagine anybody ever doing this instead of using the separate files.  I've updated the text to make this clearer in 2fe16aa9848d7bed601255980936a17bd4e911c3.